scripts/publish.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+NPM_TAG="latest"
+
+if [[ ! "$GITHUB_REF" =~ ^v\d+\.\d+\.\d+$ ]]; then
```

Comment:
We use a different regex in deploy-docs to detect the version (looking for `-` to indicate a pre-release). It's probably worth updating this to match to keep it consistent.

Reply:
I think you could use that: https://github.com/actions/upload-release-asset